melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/X006FjernInstitusjon.java

```diff
@@ -0,0 +1,13 @@
+package no.nav.melosys.eessi.models.sed.nav;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import no.nav.melosys.eessi.integration.eux.rina_api.dto.Institusjon;
```

Comment:
`Institusjon`-klassen er en dto mot eux-rina-api, og bør ikke blandes inn i det som har med melding vi legger videre på Kafka.

Reply:
Jeg ser personlig et behov for å ha intern dokumentasjon rundt klasser, etc.  Har vi det? Skal vi lage det kanskje?